.allow_skipping_tests

```diff
@@ -39,6 +39,7 @@ policies/note_policy.rb
 presenters/base_presenter.rb
 presenters/case_contact_presenter.rb
 services/case_contacts_export_csv_service.rb
+services/case_contacts_export_excel_service.rb
```

Comment:
Hmm I am not a fan of skipping tests...

Reply:
I agree, but the methods are hard to test in isolation, and you can test them in the system spec.